samples/public/BlankUwpNet9App/App.xaml.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.Storage;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace BlankUwpNet9App
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : Application
+    {
+        /// <summary>
+        /// Initializes the singleton application object. This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            InitializeComponent();
+
+            Suspending += OnSuspending;
+        }
+
+        /// <inheritdoc/>
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
+        {
+            Debugger.Launch();
+
+            Frame rootFrame = Window.Current.Content as Frame;
+
+            // Do not repeat app initialization when the Window already has content,
+            // just ensure that the window is active.
+            if (rootFrame == null)
+            {
+                // Create a Frame to act as the navigation context and navigate to the first page
+                rootFrame = new Frame();
+
+                rootFrame.NavigationFailed += OnNavigationFailed;
+
+                if (e.PreviousExecutionState == ApplicationExecutionState.Terminated)
+                {
+                    // TODO: Load state from previously suspended application
+                }
+
+                // Place the frame in the current Window
+                Window.Current.Content = rootFrame;
+            }
+
+            Microsoft.VisualStudio.TestPlatform.TestExecutor.UnitTestClient.CreateDefaultUI();
```

Comment:
Is this doing anything? 😕 

https://github.com/microsoft/vstest/blob/06f4f131ae42e4039e48924b9e8c9dbc4e3ec1b9/src/testhost.x86/UnitTestClient.cs#L15-L20

Reply:
Not as far as I could tell. But all the examples use it...

I'll leave this unresolved, you can figure out if it needs to be deprecated.

Reply:
I'll resolve this. This is indeed in all examples of VSTest UI (uwp, winui...).